Whisky/Utils/Winetricks.swift

```diff
@@ -11,6 +11,7 @@ import AppKit
 class Winetricks {
     static func isWinetricksInstalled() -> Bool {
         return FileManager.default.fileExists(atPath: "/opt/homebrew/bin/winetricks")
+               || FileManager.default.fileExists(atPath: "/usr/local/bin/winetricks")
```

Comment:
This would break Winetricks for ARM Macs? 

I'd suggest implementing a function that discovers winetricks' install location instead.

Reply:
@Youssef-Azijni It just be compatible with Intel Macs. And in swift, calling command line then get its output is suck, so I don't want do it by calling `which winetricks`